src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -16,8 +23,10 @@ import {
   NavigationHeaderDropdownMenuProps,
   SupportedIcon,
 } from 'components/navigation-header/types'
+
+// Local imports
 import s from './dropdown-menu.module.css'
-import useOnFocusOutside from 'hooks/use-on-focus-outside'
+import classNames from 'classnames'
```

Comment:
Not sure if you want to move this one up with your third-party imports section 😄 I like the idea of the comments to organize

Reply:
Oops! Yes definitely 😁

Reply:
Fixed in: https://github.com/hashicorp/dev-portal/pull/280/commits/c84c79c4fa3baad9fb641f43054f0523049cad64 ✅